composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,32 @@
+from google.cloud import bigquery
+import google.cloud.exceptions
+import pytest
+import uuid
+import os
+from . import dataflowtemplateoperator_create_dataset_and_table_helper as helper
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+
+client = bigquery.Client()
+
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
+
+expected_schema = schema = [
+    bigquery.SchemaField("location", "GEOGRAPHY", mode="REQUIRED"),
+    bigquery.SchemaField("average_temperature", "INTEGER", mode="REQUIRED"),
+    bigquery.SchemaField("month", "STRING", mode="REQUIRED"),
+    bigquery.SchemaField("inches_of_rain", "NUMERIC", mode="NULLABLE"),
+    bigquery.SchemaField("is_current", "BOOLEAN", mode="NULLABLE"),
+    bigquery.SchemaField("latest_measurement", "DATE", mode="NULLABLE"),
+]
+
+def test_creation():
+    dataset = helper.create_dataset(PROJECT_ID, dataset_UUID)
+    table = helper.create_table(PROJECT_ID, dataset_UUID)
+
+    assert table.table_id == "average_weather"
+    assert dataset.dataset_id == dataset_UUID
+    assert table.schema == expected_schema
+
+    client.delete_dataset(dataset, delete_contents=True, not_found_ok=True)
```

Comment:
Can you put those `delete` calls in `finally` clause so that they will always run?